### PR TITLE
Solved #32, added 'How to Use ?'

### LINKED ISSUE #32: Write a "How to Use?" section for README.

### PR DESCRIPTION
**Pull request by Sai-Adarsh**
Fixes #32. added 'How to use'

-----
[View rendered README.md](https://github.com/Sai-Adarsh/HackerRank-Test-Case-Generator/blob/master/README.md)

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Hi! Thanks for PR.
Make sure you have checked points in PR Template correctly.

(Powered by [GitMate.io](https://gitmate.io))

File: README.md [hunk 1]
```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
...
+```sh
+   $ source ./venv/bin/activate
+```
+* #### Activate virtualenv (Windows)
+```sh
+   $ cd venv/Scripts/
+   $ activate
+   $ pip install -r requirements.txt
+```   
+* Go to ```sh tc_generator ``` and run the project.
```
Comment (aashutoshrathi) [inline README.md]: Remove `sh` from this line, use single backtick here

File: README.md [hunk 2]
```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
 ------------------ | ------------- |
 C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
+## Install
+
+### How to Use ? 😃
+
+* Clone the repository ```sh $ git clone https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator.git ```.
+
+* Create a virtual environment ```sh $ virtualenv venve ```,
```
Comment (aashutoshrathi) [inline README.md]: Same here also remove `e` from end.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
 ------------------ | ------------- |
 C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
+## Install
+
+### How to Use ? 😃
+
+* Clone the repository ```sh $ git clone https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator.git ```.
```
Comment (aashutoshrathi) [inline README.md]: Same here,
Also, URL is incorrect.

**Comment by aashutoshrathi** [comment]
Also, squash your commits

File: README.md [hunk 4]
```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
 ------------------ | ------------- |
 C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
+## Install
+
+### How to Use ? 😃
+
+* Clone the repository `$ git clone https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator.git `.
```
Comment (aashutoshrathi) [inline README.md]: This is main repo, remove `NJACKWinterofCode`

Comment (Sai-Adarsh) [reply README.md]: > This is main repo, remove `NJACKWinterofCode`

gave PR to NWoC directory

Comment (aashutoshrathi) [reply README.md]: No issues, use the real link anyways.

Comment (Sai-Adarsh) [reply README.md]: > No issues, use the real link anyways.

@aashutoshrathi giving a new PR, closing this.

**Pull request closed without merging**